digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala

```diff
@@ -235,10 +235,46 @@ class RoadAddressService(roadLinkService: RoadLinkService, eventbus: DigiroadEve
 
   def setRoadAddressFloating(ids: Set[Long]): Unit = {
     withDynTransaction {
-      ids.foreach(id => RoadAddressDAO.changeRoadAddressFloating(float = true, id))
+      // TODO: add geometry if it is somehow available
+      ids.foreach(id => RoadAddressDAO.changeRoadAddressFloating(float = true, id, None))
     }
   }
 
+  /*
+    Kalpa-API methods
+  */
+
+  def getRoadAddressesLinkByMunicipality(municipality: Int): Seq[RoadAddressLink] = {
+    val linkIdsAmmounts = 100
+    var roadLinks = roadLinkService.getViiteRoadLinksFromVVHByMunicipality(municipality)
```

Comment:
Avoid the use of the "var" we can have one val for roadlinks and another for filtered roadlinks